tendrl/monitoring_integration/grafana/create_alert_dashboard.py

```diff
@@ -1,97 +1,339 @@
-import etcd
+import copy
```

Comment:
Name of source looks to me more like a class name but it has got all utility functions.

Reply:
i will change it